hubspot/hubspot.go

```diff
@@ -87,7 +105,7 @@ func (c *Client) do(req *http.Request, out any) error {
 	}
 	defer resp.Body.Close()
 
-	if resp.StatusCode != http.StatusOK {
+	if resp.StatusCode < http.StatusOK || resp.StatusCode > http.StatusNoContent {
```

Comment:
hm, I understand this condition but its look strange as for me, could we change it?